data/preloaded/C/basic io.txt

```diff
@@ -0,0 +1,43 @@
+/l
+C
+/l
+
+/d
+
+There are several input and output throughputs in a C program.
```

Comment:
Might want to append a "Contents: " cos the parser currently looks out for it bfore parsing the language